### PR TITLE
Add support for CanadaPostPws

### PR DESCRIPTION
**Pull request by fredericboivin**
CanadaPost is the legacy API

CanadaPostPws is the currently supported API

Some services might be missing, I added the ones currently used/supported by ActiveShipping

### CODE REVIEW

File: spec/support/shared_contexts/package_setup.rb [hunk 1]
```diff
@@ -32,3 +32,73 @@
   end
   let(:package) { order.shipments.first.to_package }
 end
+
+shared_context 'Canada package setup' do
```
Comment (forkata) [inline spec/support/shared_contexts/package_setup.rb:36]: Are these going to be used anywhere in the specs? I like the idea, but a bit hesitant to include here unless they are used.

**Comment by forkata** [comment]
@dangerdogz Thanks for adding this! That first commit is a bit large but adding support for this is awesome! We can finally look at deprecating the old Canada Post calculators :+1:

File: spec/support/shared_contexts/package_setup.rb (hunk 1 above)
Comment (fredericboivin) [reply spec/support/shared_contexts/package_setup.rb:36]: It is used in the CanadaPostPws spec to get domestic rates

**Comment by fredericboivin** [comment]
I will split it in smaller commits

**Review by forkata — APPROVED** [review]

**Pull request merged**